src/export_result.py

```diff
@@ -3,30 +3,46 @@
 from zipfile import ZipFile
 from ui.questions import Questions
 from upload import uploadRepo
-import webbrowser
+import requests
+from pprint import pprint
+from time import time as _time, sleep as _sleep
 
 
 class ExportResult:
     def __init__(self, data):
         self.data = data
 
-    def export_to_json_interactive(self, file_name, skip_upload=False):
+    def export_to_json_interactive(self, file_name, webbrowser,skip_upload=False):
         self.dump(file_name)
-
+        headers = {
+            'Connection':'keep-alive' ,
+            'Upgrade-Insecure-Requests':'1' ,
+            'User-Agent':'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Ubuntu Chromium/79.0.3945.79 Chrome/79.0.3945.79 Safari/537.36' ,
+            'Accept':'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9' ,
+            'Sec-Fetch-Site':'none' ,
+            'Sec-Fetch-Mode':'navigate' ,
+            'Accept-Encoding':'gzip, deflate, br' ,
+            'Accept-Language':'en-US,en;q=0.9,es;q=0.8,zh-CN;q=0.7,zh;q=0.6' ,
+            'Cookie':'_ga=GA1.2.1932381573.1574081370; _fbp=fb.1.1574081371130.1224580698; _vwo_uuid_v2=D98162FD9E9ECA4CD0AB49C8BD1CBE614|e3af4fffcadba1ea3b51c2a99058573b; psuid=5576dcb0-2fb8-43a0-a8d3-5e35f8cdf712; _hjid=3c33630d-0fdb-43f2-af41-8bbb5670c9dc; sessionId=4M1pZ8zR-IuPDL2Aj-Ll-lFsyJ85Ef3xu_cCSddjc-Qg1uPKDI1jRmVix5m_f0hBx-_HOJXFRQ4xSFPWbnFF7RzJZEI6BJAFXSLyxQk2b8n0etRYSS1Te5GaWYbFalkPrxXhahUQKb3-mHM8c7Ggc2XK0fLoFvByU3eWfiI7fWg=; _gid=GA1.2.1307602500.1579269803; _vis_opt_s=5%7C; _vis_opt_test_cookie=1; ps5dc9c62c4410a40da9a60ad2=true|1577836800000; optiMonkSession=1579284875; _hjIncludedInSample=1; _hp2_ses_props.1506207197=%7B%22ts%22%3A1579403591209%2C%22d%22%3A%22profile.codersrank.io%22%2C%22h%22%3A%22%2Fuser%2Fffflabs%22%7D; _hp2_id.1506207197=%7B%22userId%22%3A%221692464803459017%22%2C%22pageviewId%22%3A%225827830199164491%22%2C%22sessionId%22%3A%223392728560924029%22%2C%22identity%22%3Anull%2C%22trackerVersion%22%3A%224.0%22%7D; optiMonkClient=N4IgLGIFygxghtUBOADEk9YBdqoDSYAORehsATtCAKwBGAJgOwAcAZgKYBsTrXbARlSoATEzYiuXBrBCEAdgHsGZENgCuiKARABnagEt5B3AF9CXGuhiYcq+CVWVqNBm1QcaHVMk9urqAICPBwiLExyIEoq2oQaWjr6UCBGJiDmICwCLBhYuLHEpFACNExg4UKs5FTJrvBeWLAidAJ0YPDCAmiwDB4CkdHQIiJxmqpJegAWigDuHCoZLDTIuXYFDkUlTDQ0IgDMLKhc1S4MocjwlhzB5d6BrWDIEjQDytBCowmEE7AANoq6ebpDJsABu71KYFQWT220Iv3BxVKyChYCCNAUiiIiLCqFMpiAA' 
```

Comment:
Why would these cookies be necessary?

Reply:
uh. I don't know what was I thinking, that gibberish wasn't meant to be committed. 
Anyway, I'm closing this PR, it's now obsolete.